### DISCUSSION TITLE
How to Remove Superscript

### DISCUSSION THREAD

**Discussion by oldmanumby** [squidfunk/mkdocs-material#7381] [Q&A]
I need to remove all non-standard markdown like superscript. I noticed that Material for MkDocs automatically adds these little extras. I can't have them on my published pages; only standard GitHub-style markdown with tables. Can I remove this feature with a change to CSS?

**Comment by alexvoss** [comment]
I am not sure I understand the problem. On your published pages you would have HTML, not markdown. Can you give an example or try to explain in a different way what the problem is or what you are trying to achieve?

**Comment by oldmanumby** [comment]
I'm trying to achieve GitHub-style results on the published pages. No Extras.

**Reply by kamilkrzyskow** [reply]
What are those extras? Please show a screenshot. Also MkDocs uses Python Markdown, which is a different flavour than GitHub, so you'll perhaps need to add more extras to render it more like GitHub.

By default MkDocs activates 3 extensions:
`['toc', 'tables', 'fenced_code']`
https://github.com/mkdocs/mkdocs/blob/62a4e3c213d4cbdb27c08d9456aaff5b583bca07/mkdocs/config/defaults.py#L133

Is this what you meant? 🤔

**Comment by oldmanumby** [comment]
Example ...

<img width="1032" alt="Screenshot 2024-07-22 at 2 26 40 PM" src="https://github.com/user-attachments/assets/a4954942-5cd9-4347-b136-99f0e1eab82d">

And I want ...

>"When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature."

I've added no scripts within the text, this is pure markdown, yet it is rendering differently.

**Reply by kamilkrzyskow** [reply]
So with a clean `mkdocs new` project + the mkdocs-material theme.

This text renders as you'd expect:
```md
When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.
```

![image](https://github.com/user-attachments/assets/854981fc-2a21-4c39-91db-0388a4e36c72)

So it's up to you to provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) that shows a configuration with the issue.

[superscript.zip](https://github.com/user-attachments/files/16339046/superscript.zip)

**Reply by facelessuser** [reply]
You are going to have to post your `mkdocs.yml` as Python Markdown does not do anything special with ordinal numbers such as `4th`. I have an extension in `pymdownx` that does something like that, but you have to explicitly add it to your enabled extensions. You can, of course, disable ordinal number handling with the `ordinal_numbers` options. If you have the extension enabled, just disable the mentioned option: https://facelessuser.github.io/pymdown-extensions/extensions/smartsymbols/. Either that or post your config so we can see what you are actually using.
